subprojects/docs/src/snippets/files/copy/kotlin/build.gradle.kts

```diff
@@ -12,13 +12,6 @@ tasks.register<Copy>("copyReport") {
 }
 // end::copy-single-file-example[]
 
-// tag::copy-single-file-example-without-file-method[]
```

Comment:
❌ This removal breaks https://github.com/gradle/gradle/blob/f683f626168a7254a209976eb090646ba812fd56/subprojects/docs/src/docs/userguide/authoring-builds/working_with_files.adoc?plain=1#L44

Reply:
I removed that content as well